File: ClinicDesktop/untitled/src/com/example/clinicsystem/code/models/classes/M_EmergencyContact.java
package com.example.clinicsystem.code.models.classes;

public class M_EmergencyContact {
    private int contactID, patientID;
    private String name, relation, phoneNumber;

    public M_EmergencyContact(int contactID, int patientID, String name, String relation, String phoneNumber) {
        this.contactID = contactID;
        this.patientID = patientID;
        this.name = name;
        this.relation = relation;
        this.phoneNumber = phoneNumber;
    }

    public M_EmergencyContact() {
    }

    public int getContactID() {
        return contactID;
    }

    public void setContactID(int contactID) {
        this.contactID = contactID;
    }

    public int getPatientID() {
        return patientID;
    }

    public void setPatientID(int patientID) {
        this.patientID = patientID;
    }

    public String getName() {
        return name;
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getRelation() {
        return relation;
    }

    public void setRelation(String relation) {
        this.relation = relation;
    }

    public String getPhoneNumber() {
        return phoneNumber;
    }

    public void setPhoneNumber(String phoneNumber) {
        this.phoneNumber = phoneNumber;
    }
}

File: clinic/M_EmergencyContact.php
<?php
class M_EmergencyContact {
	private $contactID, $patientID;
    private $name, $relation, $phoneNumber;

    public function __construct($contactID, $patientID, $name, $relation, $phoneNumber) {
        $this->contactID =   $contactID;
        $this->patientID =   $patientID;
        $this->name =        $name;
        $this->relation =    $relation;
        $this->phoneNumber = $phoneNumber;
    }

    public function getContactID() {
        return $this->contactID;
    }

    public function setContactID($contactID) {
        $this->contactID = $contactID;
    }

    public function getPatientID() {
        return $this->patientID;
    }

    public function setPatientID($patientID) {
        $this->patientID = $patientID;
    }

    public function getName() {
        return $this->name;
    }

    public function setName($name) {
        $this->name = $name;
    }

    public function getRelation() {
        return $this->relation;
    }

    public function setRelation($relation) {
        $this->relation = $relation;
    }

    public function getPhoneNumber() {
        return $this->phoneNumber;
    }

    public function setPhoneNumber($phoneNumber) {
        $this->phoneNumber = $phoneNumber;
    }
}

?>